pkg/runtime/wfengine/backends/actors/activity_actor.go

```diff
@@ -203,7 +203,8 @@ func (a *activityActor) executeActivity(ctx context.Context, name string, eventP
 	//       introduce some kind of heartbeat protocol to help identify such cases.
 	callback := make(chan bool)
 	wi.Properties[CallbackChannelProperty] = callback
-	wfLogger.Debugf("Activity actor '%s': scheduling activity '%s' for workflow with instanceId '%s'", a.actorID, name, wi.InstanceID)
+	activityInfo := fmt.Sprintf("reminder '%s' for activity [%s]", name, activityName)
```

Comment:
Info level should be reserved for non-frequent information events which a human operator would be interested in in order to understand how the system is behaving. These logs don’t fit that, and similarly, is going to result in Gb of log data for any non-trivial workflow deployment quickly. Logs like these are also not an appropriate format/stream for doing observability on. This should remain in Debug.

I understand the necessity for needing to get workflow information, however as you stated, this should come from a tracing system or, ideally, an event stream which is not yet implemented.

if we really must have this, please, make this configurable with a CLI flag that only enables the log in workflows, is marked as deprecated&temporary with a warning of the volume it will produce. It should be false by default.

Reply:
+1

Reply:
will add a cli flag